config/local_restrictions.yml

```diff
@@ -179,105 +179,111 @@ E06000030:
 E06000031:
   name: Peterborough City Council
   restrictions:
-    - alert_level: 2
-      start_date: 2020-12-02
-      start_time: "00:01"
     - alert_level: 3
       start_date: 2020-12-19
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
```

Comment:
I think this needs to be `2020-12-20`

Reply:
🤦 I copied it from an earlier notes file after my editor crashed. Replaced them all now